translations/de.ts

```diff
@@ -1037,7 +1041,7 @@ Disabling chat history now will leave the encrypted history intact (but not usab
         <location filename="../src/widget/form/settings/generalsettings.ui" line="119"/>
         <source>qTox will start minimized in tray.</source>
         <comment>toolTip for Start in tray setting</comment>
-        <translation>qTox minimiert starten.</translation>
+        <translation>qTox wird minimiert starten.</translation>
```

Comment:
Nein. Der Originaltext, sagt dies: "qtox will start minimized in tray", nicht "Start qTox minimized in tray". Außerdem erklären die Tooltips normalerweise näher, weshalb sie gewöhnlich die Auswirkung der Einstellung erläutern.